src/interfaces/coral_web/src/components/Configuration/Tools.tsx

```diff
@@ -95,14 +95,14 @@ const ToolSection = () => {
             />
           </div>
           <div className="flex flex-col gap-y-5">
-            {tools.map(({ name }) => {
+            {tools.map(({ name, error_message }) => {
               const enabledTool = enabledTools.find(
                 (enabledTool) => enabledTool.name.toLocaleLowerCase() === name.toLocaleLowerCase()
               );
               const checked = !!enabledTool;
 
               return (
-                <Fragment key={name}>
+                <div key={name} className="flex gap-x-1">
                   <Checkbox
                     checked={checked}
                     onChange={async (e) => {
```

Comment:
not part of this but I don't think we need this `async`
```suggestion
                    onChange={(e) => {
```